integration-tests/src/test/java/org/openmrs/module/fhir2/BaseFhirIntegrationTest.java

```diff
@@ -86,6 +87,7 @@ public abstract class BaseFhirIntegrationTest<T extends IResourceProvider, U ext
 	
 	@Before
 	public void setup() throws Exception {
+		FhirGlobalPropertyHolder.reset();
```

Comment:
This was ultimately the key important fix, once I made it to the integration tests.